### PR TITLE
fix: shared configs/fix jest configs CME-51

### PR DESCRIPTION
**Pull request by pierrezimmermannbam**
### Context

This pr proposes a few fixes to be able to use the jest configs in kitt-universal

### CODE REVIEW

File: @ornikar/jest-config-react-native-web/package.json [hunk 1]
```diff
@@ -12,12 +12,14 @@
     "access": "public"
   },
   "peerDependencies": {
-    "react-native-web": "^0.17.5"
+    "react-native-web": "^0.17.5",
+    "jest": ">=24.0.0"
```
Comment (christophehurpeau) [inline @ornikar/jest-config-react-native-web/package.json:16]:
```suggestion
    "jest": ">=26.6.3"
```

on doit etre en 26 partout

File: @ornikar/jest-config-react/package.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
     "@testing-library/react": "^11.2.5 || ^12.0.0",
     "@testing-library/react-hooks": "^5.1.0 || ^7.0.0",
     "react": "^16.6.3 || ^17.0.0",
-    "react-dom": "^16.6.3 || ^17.0.0"
+    "react-dom": "^16.6.3 || ^17.0.0",
+    "jest": ">= 24.0.0"
```
Comment (christophehurpeau) [inline @ornikar/jest-config-react/package.json:19]:
```suggestion
    "jest": ">= 26.6.3"
```

File: @ornikar/jest-config/package.json [hunk 3]
```diff
@@ -20,5 +20,8 @@
   },
   "peerDependencies": {
     "jest": ">= 24.0.0"
```
Comment (christophehurpeau) [inline @ornikar/jest-config/package.json:22]:
```suggestion
    "jest": ">= 26.6.3"
```

Comment (pierrezimmermannbam) [reply @ornikar/jest-config/package.json]: c'est corrigé

File: @ornikar/jest-config-react-native-web/package.json (hunk 1 above)
Comment (pierrezimmermannbam) [reply @ornikar/jest-config-react-native-web/package.json]: fixed

**Review by christophehurpeau — APPROVED** [review]

**Pull request merged**